planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/utils/utils.cpp

```diff
@@ -35,6 +35,8 @@
 #include <autoware_lanelet2_extension/utility/utilities.hpp>
 #include <autoware_vehicle_info_utils/vehicle_info.hpp>
 #include <range/v3/algorithm/any_of.hpp>
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/transform.hpp>
```

Comment:
can't you just `#include <ranges>`

Reply:
ranges library only available from c++20.
non c++20 uses range-v3.
also if we include all, potentially it will increase build time, so it might be better to just include what we use.